roles/kuryr_master/tasks/main.yaml

```diff
@@ -0,0 +1,52 @@
+---
+- name: Perform OpenShit ServiceAccount config
+  include: serviceaccount.yaml
+
+- name: Create kuryr manifests tempdir
+  command: mktemp -d
+  register: manifests_tmpdir
+
+- name: Create kuryr ConfigMap manifest
+  become: yes
+  template:
+    src: configmap.yaml.j2
+    dest: "{{ manifests_tmpdir.stdout }}/configmap.yaml"
+
+- name: Create kuryr-controller Deployment manifest
+  become: yes
+  template:
+    src: controller-deployment.yaml.j2
+    dest: "{{ manifests_tmpdir.stdout }}/controller-deployment.yaml"
+
+- name: Create kuryr-cni DaemonSet manifest
+  become: yes
+  template:
+    src: cni-daemonset.yaml.j2
+    dest: "{{ manifests_tmpdir.stdout }}/cni-daemonset.yaml"
+
+- name: Apply ConfigMap manifest
+  oc_obj:
+    state: present
+    kind: ConfigMap
+    name: "kuryr-config"
+    namespace: "kube-system"
+    files:
+    - "{{ manifests_tmpdir.stdout }}/configmap.yaml"
```

Comment:
Nit:  we should use the file extension '.yml' instead of '.yaml'.  This applies in various places in the PR.

Reply:
Any pointer why is that the rule? The only official resource I was able to find [1] actually advocates to use .yaml.
[1] http://www.yaml.org/faq.html

Reply:
Looks like we use a mix in this project.  Feel free to disregard this.